.travis.yml

```diff
@@ -60,11 +65,26 @@ install:
   - tar xJf autoconf-archive-2017.09.28.tar.xz
   - cp autoconf-archive-2017.09.28/m4/ax_code_coverage.m4 m4/
   - pip install --user cpp-coveralls
+# Trusty's python version is 3.4.x. Meson 0.45++ require python 3.5++
+  - pip3 install --user meson==0.44.1
+# Trusty's ninja is too old
+  - wget https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip
+  - sha256sum ninja-linux.zip | grep -q d2fea9ff33b3ef353161ed906f260d565ca55b8ca0568fa07b1d2cab90a84a07
+  - unzip ninja-linux.zip
 
 before_script:
   - ./bootstrap
 
 script:
+# if we are building with meson, only the next 5 lines are executed
```

Comment:
This comment does not seem to be accurate--it is more than 5 lines that are executed for Meson builds.